types/Shipment/Shipment.d.ts

```diff
@@ -288,4 +288,12 @@ export declare class Shipment implements IShipment {
    * @param amount
    */
   public insure(amount: number): Promise<Shipment>;
+
+  /**
+   * Generate a specific type of form for the Shipment, using the provided form options.
+   *
+   * @param formType The type of form to generate.
+   * @param formOptions Additional options to use when generating the form.
+   */
+  public generateForm(formType: string, formOptions: object | null): Promise<Shipment>;
```

Comment:
```suggestion
  public generateForm(formType: string, formOptions?: object): Promise<Shipment>;
```

Use `?` to indicate the parameter variable is optional.

https://www.educative.io/answers/how-to-use-a-question-mark-in-typescript-variables